erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md

```diff
@@ -4,30 +4,46 @@
 In ERPNext, you can mark your incoming or outgoing products for Quality
 Inspection.
 
-To enable this feature go to:
-> Home > Stock > Tools > Quality Inspection 
+To access this feature go to:
+> Home > Stock > Tools > Quality Inspection
 
 ## 1. Prerequisites
-Before creating and using a Quality Inspection, it is advised that you create the following first:
+Before creating and using a Quality Inspection, it is advised that you do the following first:
 
-* [Item](/docs/user/manual/en/stock/item)
-* Enable Quality Inspection Criteria in the Item master:
-    ![Enable Quality Inspection](/docs/assets/img/stock/enable-quality-inspection.png)
+* **Create an [Item](/docs/user/manual/en/stock/item)**.
+* **Enable Quality Inspection Criteria in the Item master**. On enabling either checkboxes, **submission** of a stock delivery/receipt document will be allowed only after a Quality Inspection is done against it:
+    ![Enable Quality Inspection](/docs/assets/img/stock/quality-inspection-pre-requisite.png)
+* (Optional) **Create a Quality Inspection Template**. You can add inspection parameters and acceptance criteria in the template, which can be easily fetched into any Quality Inspection. After saving the template, you can set this template in the Item Master (as shown above).
```

Comment:
you can set this template in the Item Master (as shown above).

where above?

Reply:
@michellealva there is a screenshot right above it

Reply:
@marination doesn't show item master.. the screenshot is of the template.. hence, confusing..

Reply:
the screenshot above is of the template and not the item master, hence it is confusing..